src/models/user.ts

```diff
@@ -86,6 +86,35 @@ const UserSchema = new mongoose.Schema({
     default: 'silver',
   },
 
+    
+  //current rd value 
+  rd: {
+    type: Number,
+    default: 350,
+  },
+  // current volatility value
+  volatility:{
+    type: Number,
+    default: 0.06,
+  },
```

Comment:
Where's the rating itself?

Reply:
Also, I think we should separate the rating schema out of the user model. A user can have a rating, or we can look up a rating later on separately by a key. Let's not have everything scattered here though.

Reply:
Need to think about it more.

Reply:
You can have a look at how the notifications model is set up and tied into this user model!